client.go

```diff
@@ -1,20 +1,37 @@
 package pack
 
 import (
+	"context"
 	"os"
 	"path/filepath"
 
+	"github.com/buildpack/imgutil"
 	dockerClient "github.com/docker/docker/client"
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/pkg/errors"
 
-	"github.com/buildpack/pack/blob"
-	"github.com/buildpack/pack/build"
-	"github.com/buildpack/pack/config"
-	"github.com/buildpack/pack/image"
+	"github.com/buildpack/pack/internal/blob"
+	"github.com/buildpack/pack/internal/build"
+	"github.com/buildpack/pack/internal/config"
+	"github.com/buildpack/pack/internal/image"
 	"github.com/buildpack/pack/logging"
 )
 
+type ImageFetcher interface {
```

Comment:
These are missing their mock generation comments.

Reply:
They're in `interfaces.go`. Happy to fix, though this was an issue before this PR